Marlin/Configuration_adv.h

```diff
@@ -1492,4 +1492,38 @@
                                          // tweaks made to the configuration are affecting the printer in real-time.
 #endif
 
+
+// some MK2_MUXER requirements
+#if ENABLED(MK2_MULTIPLEXER)
+
+  #undef DISABLE_INACTIVE_EXTRUDER
+  #define DISABLE_INACTIVE_EXTRUDER false // Keep only the active extruder enabled. MK2_MULTIPLEXER won't work otherwise.
+
+  // all extruders use E0 stepper
+  #undef E1_STEP_PIN                  
+  #undef E1_DIR_PIN
+  #undef E1_ENABLE_PIN
+
+  #undef E2_STEP_PIN                  
+  #undef E2_DIR_PIN
+  #undef E2_ENABLE_PIN
+
+  #undef E3_STEP_PIN                  
+  #undef E3_DIR_PIN
+  #undef E3_ENABLE_PIN
+
+  #define E1_STEP_PIN     E0_STEP_PIN
+  #define E1_DIR_PIN      E0_DIR_PIN
+  #define E1_ENABLE_PIN   E0_ENABLE_PIN
+
+  #define E2_STEP_PIN     E0_STEP_PIN
+  #define E2_DIR_PIN      E0_DIR_PIN
+  #define E2_ENABLE_PIN   E0_ENABLE_PIN
+
+  #define E3_STEP_PIN     E0_STEP_PIN
+  #define E3_DIR_PIN      E0_DIR_PIN
+  #define E3_ENABLE_PIN   E0_ENABLE_PIN
+  
+#endif
```

Comment:
This is a workaround for a bug elsewhere. When `MK2_MULTIPLEXER` is set, all E movement should be tailored for that system. This is handled by the `stepper_indirection.h` code, so let's see if it matches up…
```cpp
/**
 * Extruder indirection for the single E axis
 */
#if ENABLED(SWITCHING_EXTRUDER)
  . . .
#elif EXTRUDERS > 4
  . . .
#elif EXTRUDERS > 3
  . . .
#elif EXTRUDERS > 2
  . . .
#elif EXTRUDERS > 1
  #if ENABLED(DUAL_X_CARRIAGE) || ENABLED(DUAL_NOZZLE_DUPLICATION_MODE)
    . . .
  #else
    . . .
  #endif
#elif ENABLED(MIXING_EXTRUDER)
  . . .
#else
  #define E_STEP_WRITE(v) E0_STEP_WRITE(v)
  #if ENABLED(MK2_MULTIPLEXER)
    // Even-numbered steppers are reversed
    #define NORM_E_DIR() E0_DIR_WRITE(TEST(current_block->active_extruder, 0) ? !INVERT_E0_DIR: INVERT_E0_DIR)
    #define REV_E_DIR() E0_DIR_WRITE(TEST(current_block->active_extruder, 0) ? INVERT_E0_DIR: !INVERT_E0_DIR)
  #else
    #define NORM_E_DIR() E0_DIR_WRITE(!INVERT_E0_DIR)
    #define REV_E_DIR() E0_DIR_WRITE(INVERT_E0_DIR)
  #endif
#endif
```
The bug seems to be that if the value for `EXTRUDERS` isn't "1" the `MK2_MULTIPLEXER` code will never be applied. So that should be fixed! The correct code should be…
```cpp
/**
 * Extruder indirection for the single E axis
 */
#if ENABLED(SWITCHING_EXTRUDER)
  . . .
#elif ENABLED(MK2_MULTIPLEXER)
  #define E_STEP_WRITE(v) E0_STEP_WRITE(v)
  // Even-numbered steppers are reversed
  #define NORM_E_DIR() E0_DIR_WRITE(TEST(current_block->active_extruder, 0) ? !INVERT_E0_DIR: INVERT_E0_DIR)
  #define REV_E_DIR() E0_DIR_WRITE(TEST(current_block->active_extruder, 0) ? INVERT_E0_DIR: !INVERT_E0_DIR)
#elif EXTRUDERS > 4
  . . .
#elif EXTRUDERS > 3
  . . .
#elif EXTRUDERS > 2
  . . .
#elif EXTRUDERS > 1
  #if ENABLED(DUAL_X_CARRIAGE) || ENABLED(DUAL_NOZZLE_DUPLICATION_MODE)
    . . .
  #else
    . . .
  #endif
#elif ENABLED(MIXING_EXTRUDER)
  . . .
#else
  #define E_STEP_WRITE(v) E0_STEP_WRITE(v)
  #define NORM_E_DIR() E0_DIR_WRITE(!INVERT_E0_DIR)
  #define REV_E_DIR() E0_DIR_WRITE(INVERT_E0_DIR)
#endif
```